components/service/firefox-accounts/README.md

```diff
@@ -16,6 +23,102 @@ Use gradle to download the library from JCenter:
 implementation "org.mozilla.components:fxa:{latest-version}
 ```
 
+### Start coding
+
+> This tutorial is for version 0.15 of the FxA client.
+
+First you need some OAuth information. Generate a `client_id`, `redirectUrl` and find out the scopes for your application.
+See Firefox Account documentation for that. 
+
+Once you have the OAuth info, you can start adding `FxAClient` to your Android project.
+As part of the OAuth flow your application will be opening up a Web view or open the system browser.
```

Comment:
Should we mention Custom Tabs here explicitly? They are using the system browser too but it allows the app to customize the look and feel.